### ISSUE TITLE
Seeking new contributors and maintainers

### ISSUE THREAD

**Issue by mor10** [wprig/wprig#618]
Passionate about accessibility, modern PHP, JavaScript, CSS, and build processes? Want to help WordPress developers level up to modern best-practices and move the web forward through WordPress themes? The WP Rig team is looking for new contributors and maintainers!

**To join the team, voice your interest by responding to this ticket and we'll get in touch.**

## What's involved?
Being a Contributor or Maintainer of WP Rig means having a voice, and in the case of Maintainers, having decision-making power in how WP Rig evolves to help WordPress developers build themes for the future. See a full breakdown of what these and other roles mean and entail in the [Governance document](https://github.com/wprig/docs/blob/master/GOVERNANCE.md).

## Do I fit in?
If you work with WordPress themes in any capacity beyond installing and using them, you have a role to play in WP Rig's future. We are looking for developers, designers, UX professionals, accessibility users and providers, writers, the list goes on. If you are in any way involved in the creation or maintenance of themes, you have a place in the WP Rig contributor and/or maintainer team.

## How much work am I signing up for?
The commitment for a Contributor is whatever you define. The commitment of a Maintainer involves attending meetings and office hours as well as monitoring the goings on in issues and pull request discussions.

## Why are you looking for new contributors and maintainers?
The WP Rig project was started nearly 2 years ago and has been built out by a group of dedicated maintainers. Adoption in the WordPress community, especially in the enterprise, has been significant, but there is still room to grow. Due to the constantly evolving landscape of the web, several of our maintainers are no longer able to dedicate the necessary time to evolve and move this project forward. 

WP Rig was always a true open-source community project, and its direction and evolution is entirely decided by those who contribute. The project takes every effort to make the barriers to contribution as low as possible, and encourages the participation of marginalized groups. To protect everyone involved, WP Rig has a robust and enforced [Code of Conduct](https://github.com/wprig/docs/blob/master/CODE_OF_CONDUCT.md).

In short, WP Rig needs you to grow and evolve. This is your chance to turn your ideas and opinions about the future of the web into practice by working on a true open source project with real impact from day 1. As a bonus you get to work with likeminded people, learn the intricacies of managing an open source project with many participants, how to use Github, Slack, and other tools, and create new and lasting relationships with other designers, developers, and creators.

If this all sounds interesting to you, or you have questions, drop into the comments below and let's move the web forward together!

**Comment by AdamSoucie** [comment]
I'm in. Not sure how much time my schedule will allow, but happy to help wherever I can.

**Comment by richsalvucci** [comment]
I am interested in contributing.

**Comment by Cuevas370** [comment]
I’m relatively new to working with WordPress Themes, but would love to help!

**Comment by olaleye** [comment]
I am interested.

**Comment by karunshakya** [comment]
Hi
I'm interested. I hope I am to contribute.

**Comment by prikkprikkprikk** [comment]
I am also interested. We're finally getting a chance to use WP Rig for some projects at work, so I'll probably be a bit active here too as a consequence. (Although I have a slight case of impostor syndrome … But eager to learn.)

**Comment by lowisren** [comment]
Greetings, 

I am interested in contributing to the WP Rig project. My colleagues(OIT Design - North Carolina State University) are currently using WP Rig to build a new custom theme for campus.  I would love to help in any way I can.

**Comment by melissaspiegel** [comment]
I am interested!

**Comment by heathernew09** [comment]
I am definitely interested.

**Comment by p-dhiman** [comment]
I am interested but i have very tight schedule, will chime in whenever I could

**Comment by carolinan** [comment]
I am interested because I want to learn 😁

**Comment by gchristofferson** [comment]
I would love to contribute,  possibly maintain

**Comment by mattsterp** [comment]
Hello, 
I am an intermediate front-end developer with WPRig ,PHP, and design experience.  I am available for contributing things such as writing procedures, documentation and maybe more as I gain confidence.  

Kindly, Matthew

**Comment by carolinan** [comment]
@mor10 For the next 7 months I am free to decide over my time. So, how can I help, where do I start?
Where is this project going, considering full site editing?

**Comment by justlevine** [comment]
Count me in - happy to help however I can

**Comment by mor10** [comment]
Thanks for all your responses here. I've posted an official announcement on the WP Rig blog with more info: https://wprig.io/you-wp-rig-and-the-future-of-wordpress-themes/

We will organize a meeting early in January 2020 to assemble all interested parties and stake a path into the future.

**Comment by iamchetanp** [comment]
Hey, I am interested to contribute too. Count me in! Thank you 😊

**Comment by alexwcoleman** [comment]
I'd be interested in contributing. I don't know if I have the time to be a maintainer though.

**Comment by Kallal103** [comment]
I am in, I want to learn more about the WordPress ecosystem and I think WP
rig is a very good platform for this.

**Comment by schutzsmith** [comment]
@mor10 I am interested in helping! Specifically would be interested in helping with social media, documentation, and design.

**Comment by leanderlindahl** [comment]
@mor10 :+1: I too am interested in helping out.

**Comment by justlevine** [comment]
> Thanks for all your responses here. I've posted an official announcement on the WP Rig blog with more info: https://wprig.io/you-wp-rig-and-the-future-of-wordpress-themes/
> 
> We will organize a meeting early in January 2020 to assemble all interested parties and stake a path into the future.

@mor10 Any info on if/when this meeting will happen?

**Comment by jnnrdn** [comment]
I'm interested!

**Comment by drbe4t** [comment]
I'm Interested!

**Comment by drbe4t** [comment]
Hey @mor10 I created a discord server https://discord.gg/48f8zW for this. Anyone here is welcome to join. :)

**Comment by drbe4t** [comment]
@mor10 I noticed the slack invite was missing. I know that slack is sort of the go to but I was wondering if discord might be a better option?

**Comment by megphillips91** [comment]
I'm  little late to the party, but I would love to contribute to WP Rig. This is a great service to the whole WP Community! Please send me information on how to get involved, meeting times, etc.

**Comment by sbweaver** [comment]
I'm late to the party too. I am a designer who likes to develop and wants to learn - would like to help in whatever way I can.

**Comment by megphillips91** [comment]
@mor10 @sbweaver I tried to hop into the weekly office hours last week. I may have misunderstood the timezone, but I was there alone. I have decided to use WP Rig to build a companion theme I am building which will partner my Charter Boat Bookings Plugin. In the process, I built some cool little add-ons like a social sharing class which might would be cool to include modularly / optionally into core. Right now, building in a "like" button which could be a nice companion to the sharing buttons. I enjoyed the perspective in this video about modular components: https://youtu.be/2s0vFb3Xd6w

**Comment by justlevine** [comment]
half a year later and silence... Does that mean this project is abandoned?

**Comment by heathernew09** [comment]
unfortunately, I think so. I do know in it's current state dependencies need to be updated as it fails in the dev build. I haven't had a chance to explore and submit a fix, my only exposure is the linked in learning /lynda course. I really want to use it regularly though.

**Comment by stevewestdev** [comment]
Damn this was my go to theme for custom WP builds, though I seem to be doing less and less of them (building JAMstack + WP more often). So many comments here re: maintaining yet no action?

**Comment by goldtreefrog** [comment]
What do we need? How about:

1. Time
1. Direction
1. Knowledge
1. Help
1. Teamwork

I just discovered WP Rig _tonight._ I would love to help, but my time is limited and unpredictable for the next two or three months, and I am not _yet_ a WordPress developer. I took some of Morten Rand-Hendriksen's Lynda courses in 2015-17 (along with other courses), before he came out with WP Rig, and then spent the next 3 years researching cancer for a friend and taking a Node and React Bootcamp, not developing websites at all outside of course work. My own WordPress site is years out of date...

But I would like to keep in touch with people who want to work on this project, because I am now wanting to start developing websites using WP Rig, and I want it to work! I have been interested in accessibility issues since the 1990's (yes, really!) and like the accessibility emphasis here, also that WP Rig is open source. This is a project I would be proud to work on, and I am happy to learn. I hope someone else can direct the team, at least in the beginning, because I do not feel up to it myself, especially not before August 2020. (And as far as expertise, I still consider myself a mainframe database programmer, but I'm changing that this year.)

I have noticed that other open-source projects seem to die and then it only takes one or a handful of very dedicated people to revive it. We can do this! Don't give up!

**Comment by alotoftype** [comment]
I am interested as well. How can I help @goldtreefrog I appreciate your rally

**Comment by ShawnJW** [comment]
I would be interested, hope this picks up again.

**Comment by justlevine** [comment]
Maybe it's time to fork? 

Im happy to commit code, review PRs, and write documentation, but the 'management' part is way beyond me (unit testing, code coverage, contribution workflows in github...)

@benoitchantre @ataylorme @bamadesigner @felixarntz do any of you still have interest in leading/contributing (either taking the reins from @mor10 here or via a fork ?)

**Comment by drbe4t** [comment]
@justine I think it is.

**Comment by benoitchantre** [comment]
@justlevine There will be a public meeting as soon as our agenda allows it. We will keep you informed.

**Comment by heathernew09** [comment]
Good to hear, I hope to be able to contribute to keep this moving forward.

**Comment by drozumek** [comment]
I can help as well.
I know very good CSS, vanilla JS and a little bit PHP (still learning). 
I can make a documentation as well.

**Comment by mor10** [comment]
Update: A public meeting is scheduled for **Friday August 21 at 8am PST** to discuss the future of WP Rig. The meeting will be held over Zoom and everyone is free to join. Code of Conduct applies. Zoom link will be made public in this GitHub issue 15 minutes prior to the meeting.

**Comment by schutzsmith** [comment]
Sounds great! Looking forward to it

**Comment by mor10** [comment]
Update: Here's the public registration info for the meeting:

When: Aug 21, 2020 08:00 AM Pacific Time (US and Canada) 

Register in advance for this meeting:
https://linkedin.zoom.us/meeting/register/tJ0rc-mhqj4rGtbwM7UB7NL7_bOY50qkMIJ9 

After registering, you will receive a confirmation email containing information about joining the meeting.

Please read and adhere to the [WP Rig Code of Conduct](https://github.com/wprig/docs/blob/master/CODE_OF_CONDUCT.md).

**Comment by bnwigen** [comment]
I'm still learning but I would like to contribute as well.

**Comment by mor10** [comment]
Meeting agenda posted: https://wprig.io/wp-rig-august-21-2020-meeting-agenda/

**Comment by drozumek** [comment]
I missed the meeting...
Can someone summarize the meeting? I am very interested in future of the project.

**Comment by jacklowrie** [comment]
I was unable to attend the meeting, but am interested in the future of the project and in contributing. Is there a summary or meeting notes that can be made available?

**Comment by robruiz** [comment]
Are you in the WP Rig slack? If not, please join. This is where we do all
of our organizing.

Join our workspace!
https://join.slack.com/t/wprig/shared_invite/zt-18jouazkg-KMExY7ioBU0_dRRvAeTGrQ
